rest_framework/utils/field_mapping.py

```diff
@@ -251,7 +251,9 @@ def get_relation_kwargs(field_name, relation_info):
 
     limit_choices_to = model_field and model_field.get_limit_choices_to()
     if limit_choices_to:
-        kwargs['queryset'] = kwargs['queryset'].filter(**limit_choices_to)
```

Comment:
@carltongibson cool new feature. A question: it support use a callable like django `ForeignKey.limit_choices_to`?

Reply:
@luzfcb It's Friday afternoon. 🙂 

https://github.com/django/django/blob/7c68cea7ee406fcb1b632e3a1a7f6f53e1423839/django/db/models/fields/related.py#L390-L399